components/motor/roboclaw/roboclaw.go

```diff
@@ -104,11 +118,11 @@ func newRoboClaw(deps resource.Dependencies, conf resource.Config, logger golog.
 		motorConfig.Address = 128
 	}
 
-	if motorConfig.TicksPerRotation <= 0 {
+	if motorConfig.TicksPerRotation < 0 {
```

Comment:
Is there a reason for ruling out TicksPerRotation = 0? Should we return some sort of "invalid TicksPerRotation" error if that's the case?

Reply:
zero is the default if the user doesn't add ticksPerRotation into the config, so using that value to assume no encoders are connected and do the correct logic in GoFor and GoTo. But I think it might make more sense to error if they put a TicksPerRotation less than zero instead of just setting it to 1.